perun-core/src/main/java/cz/metacentrum/perun/core/api/ServicesManager.java

```diff
@@ -549,10 +549,6 @@
 	 |              |       |        +-------Attrs
 	 |              |       |        +-------ChildNodes
 	 |              |       |                   +-------()
-	 |              |       |                   |        +---ChildNodes
```

Comment:
This part of JavaDoc is needed to be removed also a few lines above.

Reply:
Ok, thanks. I have also updated the javadoc of rpc.